.github/workflows/py-ci.yml

```diff
@@ -72,6 +72,8 @@ jobs:
         run: python -m pip install poetry
       - name: install deps
         run: cd testdocs && poetry install
+      - name: skip aws test
+        run: rm docs/advanced-features/athena.md
```

Comment:
If this is really required that deserves at least a comment.

Reply:
added more expansive name.